libraries/plugins/witness/witness.cpp

```diff
@@ -109,6 +109,16 @@ void witness_plugin::plugin_initialize(const boost::program_options::variables_m
          _private_keys[key_id_to_wif_pair.first] = *private_key;
       }
    }
+   if(options.count("required-participation"))
+   {
+       auto required_participation = options["required-participation"].as<uint32_t>();
+       FC_ASSERT(required_participation < 100);
```

Comment:
Why must `<100`?

Reply:
Basically because the description says 0-99 at https://github.com/bitshares/bitshares-core/blob/eb4e92e33ca3f5957bf7b13dd7493be83a8c1129/libraries/plugins/witness/witness.cpp#L67

Reply:
I still didn't get why it should be within [0,99].

Reply:
Should we include 100 and change the description in your opinion ?

Reply:
Yes

Reply:
changed at https://github.com/bitshares/bitshares-core/pull/1787/commits/0bdf4d6d2b53305b05f11a0493162bc27a0bd49a